.travis.yml

```diff
@@ -10,8 +13,10 @@ env:
   - TEST_SUITE=./spec/build_spec.cr
   - TEST_SUITE=./spec/amber
 
-script: 
-  - docker-compose run --service-ports spec bash -c "cd /opt/amber && crystal spec $TEST_SUITE -D run_build_tests"
+script:
+  - crystal spec
+  - bin/ameba
```

Comment:
change this line to:
```
- docker-compose run bash -c "bin/ameba"
```
This will launch a bash shell on the docker image and run `bin/ameba`.  I'm assuming here that ameba gets built and installed in the project bin directory when we run `shards install`.  The default working directory is set in the Docker file and in the docker-compose.yml so we don't need to cd into the directory anymore.